panimg/image_builders/metaio_utils.py

```diff
@@ -189,23 +198,45 @@ def validate_and_clean_additional_mh_headers(
     for key, value in headers.items():
         if key in EXPECTED_HEADERS:
             cleaned_headers[key] = value
-        else:
-            if key in ADDITIONAL_HEADERS:
-                match_pattern = ADDITIONAL_HEADERS[key]
-                if not re.match(match_pattern, value):
-                    raise ValueError(
-                        f"Invalid data type found for "
-                        f"additional header key: {key}"
-                    )
-                cleaned_headers[key] = value
-        if key in HEADERS_MATCHING_NUM_TIMEPOINTS:
-            validate_list_data_matches_num_timepoints(
-                headers=headers, key=key, value=value
-            )
+        elif key in ADDITIONAL_HEADERS:
+            match_pattern = ADDITIONAL_HEADERS[key]
+            if not re.match(match_pattern, value):
+                raise ValueError(
+                    f"Invalid data type found for "
+                    f"additional header key: {key}"
+                )
+            if key in HEADERS_MATCHING_NUM_TIMEPOINTS:
+                validate_list_data_matches_num_timepoints(
+                    headers=headers, key=key, value=value
+                )
+            if key in HEADERS_MATCHING_WINDOW_SETTINGS:
+                validate_center_matches_width_setting(headers, key, value)
+            cleaned_headers[key] = value
 
     return cleaned_headers
 
 
+def validate_center_matches_width_setting(
+    headers: Dict[str, str], key: str, value: str
```

Comment:
Nitpick but style-wise I prefer keyword-only arguments for non-trivial functions.

Reply:
Totally not a nitpick but a core style. CIRRUS is getting to me. I'm all for keyword usage.